sites/fast-website/src/app/components/card-section/card-section.styles.ts

```diff
@@ -0,0 +1,27 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "@microsoft/fast-components";
+
+export const CardSectionStyles = css`
+    :host {
+        --flow: column;
+    }
+
+    @media screen and (max-width: 1200px) {
+        :host {
+            --flow: row;
+        }
+    }
+
+    ${display("grid")} :host {
+        grid-template-rows: repeat(4, min-content);
+        grid-auto-flow: var(--flow);
+    }
+
+    :host(:hover) ::slotted(feature-card) {
+        filter: blur(1px);
```

Comment:
I believe the blur in the figma is 2px. I know this is a tiny difference, but I want to make sure they my content and your cards have the same blur as they do in the figma.

Reply:
Yeah, in the design is says 2px but 2px blur in browser looks WAY more blurry than 2px blur looks in Figma.  I made the tweak to get the look.